README.md

```diff
@@ -258,46 +256,6 @@ This project contains a pre-configured configuration file `values.yaml`, you jus
 
 - `<your-container-registry>` - full-qualified hostname of your container registry
 - `domain`- full-qualified domain name used to access applications in your Kyma cluster
-
-#### Use API_BUSSINESS_PARTNER Remote Service (optional)
```

Comment:
Why is this removed?

Reply:
Support for connectivity requires an update. It is blocked due to [this](https://github.com/SAP/cloud-sdk-backlog/issues/809).

Reply:
Does this also apply to Java? The issue seems to refer to Cloud SDK Node.js..

Reply:
I think it does. According to [this](https://github.com/SAP/cloud-sdk-backlog/issues/809#issuecomment-1268050031) comment, some additional information is required to support the `Trusted` mode aka the mode in which Connectivity works in Kyma.

Reply:
Okay, I will clarify this also again with the Cloud SDK Java colleagues in our next sync.